app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -113,14 +117,17 @@
 
     private View contextMenuView;
 
-    private int columnWidth;
+    private int roomColumnWidth;
+    private int maxRoomColumnsVisible;
 
     private String lectureId;        // started with lectureId
 
     private Lecture lastSelectedLecture;
 
     private LectureViewDrawer lectureViewDrawer;
 
+    private Map<Integer, LectureViewColumnAdapter> adapterByRoom = new HashMap<>();
```

Comment:
:abc: I think `adapterByRoom` should be renamed to `roomIndexByAdapter`.

Reply:
The map maps Adapter instances by room indices. I could rename it to adapterByRoomIndex.

Reply:
:+1: Sound good. I used the "by" the other way around in my head. - `LecturesByRoom` -> `LecturesByRoomIndex` then, too?!

Reply:
I will rename it for now. 
In the future we should consider creating more value objects, e.g. Room (with titel, index etc.) and use this as map key.